core/audits/critical-request-chains.js

```diff
@@ -41,42 +41,42 @@ class CriticalRequestChains extends Audit {
     };
   }
 
-  /** @typedef {{depth: number, id: string, chainDuration: number, chainTransferSize: number, node: LH.Audit.Details.SimpleCriticalRequestNode[string]}} CrcNodeInfo */
+  /** @typedef {{depth: number, id: string, chainDuration: number, chainTransferSize: number, node: LH.Artifacts.CriticalRequestNode[string]}} CrcNodeInfo */
```

Comment:
This file has the only real changes. This was actually always `LH.Artifacts.CriticalRequestNode` because the input was the (computed) artifact, with `node.request` being an actual `NetworkRequest`, but previous to this PR that was actually structurally compatible with the simplified `LH.Audit.Details.SimpleCriticalRequestNode`:

https://github.com/GoogleChrome/lighthouse/blob/a2977ffecd19d58e74de4631d2a1159d3457523b/types/lhr/audit-details.d.ts#L32-L42

so the two types could be used interchangeably.

Since we don't want to change `LH.Audit.Details.SimpleCriticalRequestNode`, but `node.request` now has `networkRequestTime`, etc on it, this file needs to now actually differentiate between the two, and there needs to be a copy step from the new property names to the old property names on the audit details.

Nothing actually changes in the output (seen sample_v2.json for an example).